Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -56,51 +56,51 @@ using namespace HTMLNames;
 
 struct EntityDescription {
     const char* characters;
-    unsigned char length;
-    unsigned char mask;
+    uint8_t length;
+    EntityMask mask;
 };
 
 static const EntityDescription entitySubstitutionList[] = {
-    { "", 0 , 0 },
-    { "&amp;", 5 , EntityAmp },
-    { "&lt;", 4, EntityLt },
-    { "&gt;", 4, EntityGt },
-    { "&quot;", 6, EntityQuot },
-    { "&nbsp;", 6, EntityNbsp },
-    { "&#9;", 4, EntityTab },
-    { "&#10;", 5, EntityLineFeed },
-    { "&#13;", 5, EntityCarriageReturn },
+    { "", 0 , static_cast<EntityMask>(0) },
```

Comment:
Would it make sense to have a sentinel constant like so?

```
constexpr EntityMask EntityMaskNull = static_cast<EntityMask>(0);
```

Reply:
I think that's an interesting point — is it technically undefined behavior to static cast a raw int to an invalid `enum class`?

(Perhaps we could avoid this altogether by making `EntityDescription`'s `mask` an `OptionSet<EntityMask>`).

Reply:
Yeah, I've considered making it OptionSet as well. Maybe that's cleaner?

Reply:
Actually, using std::optional<> makes more sense because these mask values refer to a specific entity type, not a mask of entities.